src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -39,13 +41,23 @@ protected function configure()
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
                 new InputOption('label', 'l', InputArgument::OPTIONAL, 'Labels a graph'),
+                new InputOption('format', null, InputArgument::OPTIONAL, 'The dump format', 'dot'),
+                new InputOption(
```

Comment:
We don't do that in Symfony. All arguments should be on one line..

Reply:
alright, that is fixed.